diff -ur freenx.orig/nxsetup freenx/nxsetup
--- freenx.orig/nxsetup	2007-12-18 18:06:55 +0200
+++ freenx/nxsetup	2007-12-24 05:59:00 +0200
@@ -16,6 +16,8 @@
 LOCAL_USER_OPTION=""
 # luseradd / luserdel are only available on RedHat
 [ -f /etc/redhat-release ] && LOCAL_USER_OPTION="yes"
+# altlinux have /etc/redhat-release file but don't have luseradd / luserdel commands
+[ -f /etc/altlinux-release ] && LOCAL_USER_OPTION="no"
 SETUP_LOCAL_USER="no"
 CLEAN="no"
 UNINSTALL="no"
