--- nxnode.orig	2008-07-17 17:48:19 +0400
+++ nxnode	2008-07-17 17:49:51 +0400
@@ -253,6 +253,14 @@ node_start_applications()
 	. /etc/profile
 	[ "$ENABLE_SOURCE_BASH_PROFILE" = "1" -a -f ~/.bash_profile ] && . ~/.bash_profile
 
+	#fixkeyboard
+	export DISPLAY=:$display
+	FIXKBD="/etc/X11/xinit/fixkeyboard
+		/etc/X11/xinit/xinitrc-common"
+	for file in $FIXKBD ; do
+	    [ -x $file ] && $file
+	done
+
 	mkdir -p "$USER_FAKE_HOME/.nx/C-$sess_id/pids/apps/"
 
 	#
