--- node.conf.sample.orig	2008-06-14 18:32:00 +0400
+++ node.conf.sample	2008-06-14 18:32:12 +0400
@@ -439,7 +439,7 @@
 # Depending on distribution DEFAULT_X_SESSION might be /etc/X11/xdm/Xsession,
 # /etc/X11/Sessions/Xsession or /etc/X11/xinit/xinitrc
 #USER_X_STARTUP_SCRIPT=.Xclients
-#DEFAULT_X_SESSION=/etc/X11/xdm/Xsession
+DEFAULT_X_SESSION=/etc/X11/Xsession
 
 # The key that contains the name of the script that starts a KDE session.
 # It's run when a 'unix-kde' session is requested by the client.
