--- freenx-server/nxloadconfig.orig	2012-07-18 17:45:54.624000000 +0400
+++ freenx-server/nxloadconfig	2012-07-18 17:43:42.454003648 +0400
@@ -58,7 +58,7 @@
 # Where can different nx components be found
 NX_DIR=/usr
 PATH_BIN=$NX_DIR/bin # if you change that, be sure to also change the public keys
-PATH_LIB=$NX_DIR/lib
+PATH_LIB=$NX_DIR/lib/NX/lib
 NX_ETC_DIR=/etc/nxserver
 NX_SESS_DIR=/var/lib/nxserver/db
 NX_HOME_DIR=/var/lib/nxserver/home
@@ -183,7 +183,7 @@
 #JJK: added the following path referenced in nxprint
 PPD_DIR="/usr/share/cups/model" #JJK: Note /usr/share/ppd on some systems
 
-CUPS_BACKEND="/usr/lib/cups/backend"
+CUPS_BACKEND="/usr/libexec/cups/backend"
 CUPS_IPP_BACKEND="$CUPS_BACKEND/ipp"
 CUPS_DEFAULT_SOCK="/var/run/cups/cups.sock"
 CUPS_ETC="/etc/cups/"
@@ -202,15 +202,15 @@
 KILL_DEFAULT_X_WM="1"
 BOOTSTRAP_X_SESSION="0"
 USER_X_STARTUP_SCRIPT=.Xclients
-DEFAULT_X_SESSION=/etc/X11/xdm/Xsession
+DEFAULT_X_SESSION=/etc/X11/Sessions/Xsession
 COMMAND_GDM_X_SESSION="/etc/X11/gdm/Xsession custom"
 if [ ! -x "$COMMAND_GDM_X_SESSION" ]
 then
-	COMMAND_GDM_X_SESSION="/etc/X11/Xsession"
+	COMMAND_GDM_X_SESSION="/etc/X11/Sessions/Xsession"
 fi
 COMMAND_START_KDE="startkde"
 COMMAND_START_KDE4="startkde4"
-COMMAND_START_GNOME="gnome-session"
+COMMAND_START_GNOME="/etc/X11/Sessions/Gnome"
 COMMAND_START_CDE=cdwm
 COMMAND_XTERM=xterm
 COMMAND_XAUTH=/usr/bin/xauth
@@ -234,14 +234,15 @@
 COMMAND_NXSHADOWACL="$PATH_BIN/nxshadowacl"
 COMMAND_NXACL="$PATH_BIN/nxacl"
 COMMAND_NXCHECKLOAD="$PATH_BIN/nxcheckload"
-COMMAND_NXAGENT="$PATH_BIN/nxagent"
+COMMAND_NXAGENT="REAL_PATH_LIB/nxagent"
 
 # Guest directives
 ENABLE_GUEST_LOGIN="0"
-COMMAND_GUEST_LOGIN="/usr/lib/nx/guest/nxnode"
+COMMAND_GUEST_LOGIN="/usr/lib/NX/guest/nxnode"
 # Try to use protected enviroment for guest sessions
-COMMAND_GUEST_X_SESSION="/usr/share/gdm/guest-session/Xsession custom"
-if [ ! -x "/usr/share/gdm/guest-session/Xsession" ]
+COMMAND_GUEST_X_SESSION="/usr/lib/NX/guest/Gnome"
+
+if [ ! -x "/usr/lib/NX/guest/Gnome" ]
 then
 	COMMAND_GUEST_X_SESSION=$COMMAND_GDM_X_SESSION
 fi
@@ -406,7 +407,7 @@
 [ -z "$PROXY_LIBRARY_PATH" ] && PROXY_LIBRARY_PATH=$PATH_LIB
 
 [ -z "$APPLICATION_LIBRARY_PATH" ] && APPLICATION_LIBRARY_PATH=$PATH_LIB
-[ -z "$APPLICATION_LIBRARY_PRELOAD" ] && APPLICATION_LIBRARY_PRELOAD="$APPLICATION_LIBRARY_PATH/libX11-nx.so.6:$APPLICATION_LIBRARY_PATH/libXext-nx.so.6:$APPLICATION_LIBRARY_PATH/libXcomp.so.3:$APPLICATION_LIBRARY_PATH/libXcompext.so.3:$APPLICATION_LIBRARY_PATH/libXrender.so.1"
+[ -z "$APPLICATION_LIBRARY_PRELOAD" ] && APPLICATION_LIBRARY_PRELOAD="$APPLICATION_LIBRARY_PATH/libX11.so:$APPLICATION_LIBRARY_PATH/libXext.so:$APPLICATION_LIBRARY_PATH/libXcomp.so:$APPLICATION_LIBRARY_PATH/libXcompext.so:$APPLICATION_LIBRARY_PATH/libXrender.so"
 
 NX_BACKEND_VERSION=$(strings $COMMAND_NXAGENT 2>/dev/null | egrep 'NXAGENT - Version' | sed 's/.*Version //g')
 
